tests/conftest.py

```diff
@@ -4,41 +4,41 @@
 
 from requests import Response
 
+from mws.mws import MWS
 from mws.utils.xml import MWS_ENCODING
 from mws.response import MWSResponse
 
 
-@pytest.fixture
-def cred_access_key():
-    return "cred_access_key"
-
-
-@pytest.fixture
-def cred_secret_key():
-    return "cred_secret_key"
-
-
-@pytest.fixture
-def cred_account_id():
-    return "cred_account_id"
+TEST_MWS_ACCESS_KEY = "my_access_key"
+TEST_MWS_SECRET_KEY = "my_secret_key"
+TEST_MWS_ACCOUNT_ID = "my_account_id"
+TEST_MWS_AUTH_TOKEN = "my_auth_token"
```

Comment:
These *were* fixtures, but I found it cumbersome trying to access them within low-level methods that were not, themselves, tests. As they're just magic strings with some known values, I opted to change them to simple constants and import them where needed.